Userland/Libraries/LibCore/DateTime.cpp

```diff
@@ -7,6 +7,8 @@
 #include <AK/StringBuilder.h>
 #include <AK/Time.h>
 #include <LibCore/DateTime.h>
+#include <cctype>
+#include <cerrno>
```

Comment:
These are standard C++ headers, please don't include them, use the C variants

Reply:
Please use `AK/CharacterTypes.h` (which uses `constexpr` and supports full Unicode) over `ctype.h` unless you have a good reason to. (It's only been merged today, so you maybe need to rebase to get access to it.)